pay-api/tests/utilities/base_test.py

```diff
@@ -1090,7 +1090,8 @@ def factory_fee_schedule_model(
     service_fee: FeeCode = None,
     variable=False,
     show_on_pricelist=False,
-):
+    enable_gst=False,
```

Comment:
@seeker25 I saw this column was renamed, will rename this along with the copied function based on your WIP PR